Corona-Warn-App/src/main/java/de/rki/coronawarnapp/dccticketing/ui/certificateselection/DccTicketingCertificateSelectionHelperFunctions.kt

```diff
@@ -0,0 +1,28 @@
+package de.rki.coronawarnapp.dccticketing.ui.certificateselection
+
+import android.content.Context
+import de.rki.coronawarnapp.R
+import de.rki.coronawarnapp.dccticketing.core.certificateselection.DccTicketingCertificatesFilterType
+import de.rki.coronawarnapp.dccticketing.core.certificateselection.DccTicketingCertificatesFilterType.PCR_TEST
+import de.rki.coronawarnapp.dccticketing.core.certificateselection.DccTicketingCertificatesFilterType.RA_TEST
+import de.rki.coronawarnapp.dccticketing.core.certificateselection.DccTicketingCertificatesFilterType.RECOVERY
+import de.rki.coronawarnapp.dccticketing.core.certificateselection.DccTicketingCertificatesFilterType.TEST
+import de.rki.coronawarnapp.dccticketing.core.certificateselection.DccTicketingCertificatesFilterType.VACCINATION
+
+fun Context.certificateTypesText(certificateTypes: List<String>, separator: String = ", "): String =
```

Comment:
Would be better to use LazyString, so the context is only needed when accessing the String.

Reply:
this is accessed from the view holder, so it when actually used